go-app.js

```diff
@@ -12,10 +12,16 @@ go.app = function() {
     var ChoiceState = vumigo.states.ChoiceState;
     var EndState = vumigo.states.EndState;
     var FreeText = vumigo.states.FreeText;
+    var JsonApi = vumigo.http.api.JsonApi;
+    var _ = require("lodash");
 
     var GoApp = App.extend(function(self) {
         App.call(self, 'states:start');
 
+        self.init = function() {            
+        self.http = new JsonApi(self.im);       
+       };
```

Comment:
Could you indent the `self.http = ...` line one level and make the indentation of `};` match that of the `self.init = function() {` line?